### DISCUSSION TITLE
Worriend about mesh leveling

### DISCUSSION THREAD

**Discussion by Vincentbamps** [suchmememanyskill/PrusaSlicer-Ender3-v3-SE-Config#34] [Q&A]
Does PrusaSlicer, or the PrusaSlicer profile , recognize the mesh-leveling data from the V3 SE? Without this data, the printer head won’t be able to compensate for an uneven bed.

**Comment by suchmememanyskill** [comment]
@Vincentbamps I still am not entirely sure.
Assuming creality is competent, leveling is restored after G28 (home).
When i still ran stock firmware, enabling the calibration option before print also worked (suggesting that restore after home is enabled).
Otherwise you can manually set it using [M420 S1](https://marlinfw.org/docs/gcode/M420.html) or [G29](https://marlinfw.org/docs/gcode/G029-ubl.html) after G28 in the Custom Start G-Code
